Assets/HotmaxEditorVR/objectSelect.cs

```diff
@@ -5,174 +5,158 @@
 
 public class objectSelect : MonoBehaviour
 {
-    public Valve.VR.InteractionSystem.Hand hand1;
-    public Valve.VR.InteractionSystem.Hand hand2;
+    //--laser stuff
+    private Vector3 laserEndPosition;
+    public LineRenderer laserLineRenderer;
+    private float laserWidth = 0.01f;
```

Comment:
make 11 and 12 private, but add [SerializeField] for inspector

Reply:
got it!